packages/specs/source/peer-protocol.md

```diff
@@ -32,28 +30,21 @@ A protocol consists of the following components:
 
 ## Primitive Types
 
-|   Type    |          Description          |                                                      Sample                                                       |
```

Comment:
we are using this extension: https://marketplace.visualstudio.com/items?itemName=shuworks.vscode-table-formatter

please do not format it any other way